exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,10 @@
 // Start by creating a variable `message`
 
-console.log(message);
+var greetingStart = "Hello, my name is ";
+var name = "Dilek";
+
+var greeting = greetingStart + name;
+
+console.log(greeting);
+
```

Comment:
Good. It doesn't matter here but be careful about using the right variable name.